test/run_tests.py

```diff
@@ -203,12 +205,13 @@ def sauce_credentials(sauceusername, saucekey):
     )
 
     args = parser.parse_args()
-    browser = args.browser.lower()
-    if browser is not 'chrome' and env.TRAVIS_EVENT_TYPE in TRAVIS_PRS:
+    browser = args.browser.lower().strip()
+    if browser != 'chrome' and env.TRAVIS_EVENT_TYPE != 'cron':
         print(
-            'Can not run test with browser {} from SauceLabs\n'
+            'Can not run test with browser "{}" from SauceLabs\n'
             'SauceLabs can be used only when running with corn and from '
```

Comment:
Typo. it should be:
`             'SauceLabs can be used only when running with cron and from '`